packages/typescript/package.json

```diff
@@ -8,6 +8,9 @@
   "license": "MIT",
   "repository": "rollup/plugins",
   "author": "Oskar Segersvärd",
+  "contributors": [
```

Comment:
`maintainers` would be better here. we track and credit contributors here: https://github.com/rollup/plugins/blob/07d0fbf99b7cedf106b89df7961c229510893a3c/LICENSE#L3

Reply:
I'll just remove this for now and we can potentially add that field later. It seems that `maintainers` is set by NPM automatically so whatever we add could get overwritten.